.github/workflows/weekly-cea.yml

```diff
@@ -0,0 +1,49 @@
+name: Weekly CEA CI
+
+on:
+  schedule:
+    - cron: "0 2 * * 6" # every Saturday at 2am UTC
+  workflow_dispatch:
+
+jobs:
+  build_and_test:
+    strategy:
+      matrix:
+        build_type:
+          - Release
+          - Debug
+        backend:
+          - name: cuda-a100
+            flags: -DKokkos_ENABLE_CUDA=ON -DKokkos_ARCH_AMPERE80=ON
+            modules: gcc/11.2.0/gcc-4.8.5 cuda/12.2.1/gcc-11.2.0 cmake/3.28.3/gcc-11.2.0
+            queue: gpua100
+            runner: [self-hosted, cuda]
+
+    runs-on: ${{ matrix.backend.runner }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
```suggestion
      - uses: actions/checkout@v4
```
With this you already get meaningful step summary:
![image](https://github.com/user-attachments/assets/cc3fbc86-aaca-4b19-a08d-951553b11d27)

Reply:
Right.